keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone_presets_test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 The KerasCV Authors
```

Comment:
We should also have a separate `efficientnet_v2_backbone_test`, as well as one test case here which asserts the numerical results of a forward pass with a preset.

Reply:
thats done in the ImageClassifierTest.  Do you think thats sufficient?